src/YieldDistributor.sol

```diff
@@ -182,10 +186,12 @@ contract YieldDistributor is OwnableUpgradeable {
      * @return bytes Calldata used by the resolver to distribute the yield
      */
     function resolveYieldDistribution() public view returns (bool, bytes memory) {
+        uint256 _available_yield = BREAD.balanceOf(address(this)) + BREAD.yieldAccrued();
         if (
             currentVotes == 0 // No votes were cast
                 || block.number < lastClaimedBlockNumber + cycleLength // Already claimed this cycle
-                || BREAD.balanceOf(address(this)) + BREAD.yieldAccrued() < projects.length // Yield is insufficient
+                || _available_yield < projects.length // Yield is insufficient
```

Comment:
this is made redundant by the next line.

Reply:
So if yield = 9
projectsLength = 5 
yieldFixedSplit = 2 

it should pass the first line ( 9 < 5 == false) 
but fail the second like ( 9 / 2 < 5 == true) 

This is def an edge case but I'm trying to close #45

Reply:
addressed with 50c4082